EmbeddedStock/global.json

```diff
@@ -0,0 +1,6 @@
+{
+"projects": [ "src", "test" ],
```

Comment:
https://ievangelist.github.io/blog/the-global-json/
Her står der følgende om "projects":
_"The projects property is an array of strings. It’s defined as the following:
A list of project folders relative to this file."_

Vi har ikke nogen projekter der hverken hedder "src" eller "test"..